src/components/Shared/Contents/styles.js

```diff
@@ -2,9 +2,15 @@ import styled from 'react-emotion';
 
 const ContentWrapper = styled.div`
   background-color: #fff;
+<<<<<<< HEAD
   margin: 1.5rem;
   padding: 1rem;
   flex-grow: 9;
+=======
+  margin: 40px;
+  padding: 20px;
+  float: left;
+>>>>>>> Google Login update part
```

Comment:
I think this is the reason why the Table disappear...
Please delete the code below 🙇‍♂️

```
=======
  margin: 40px;
  padding: 20px;
  float: left;
>>>>>>> Google Login update part

```